demo/api/src/products/entities/product-variant.entity.ts

```diff
@@ -33,7 +34,8 @@ export class ProductVariant extends BaseEntity<ProductVariant, "id"> implements
         // search: true, // not yet supported for nested
         // filter: true, // not yet supported for nested
         // sort: true, // not yet supported for nested
-        // input: true, // not supported for nested, doesn't make sense
+        input: false, // default is true, but usually not needed if it's a rootArg
+        rootArg: true, // default is false, will create a query arg + create arg similar to scope
```

Comment:
I'm not sure about the naming... if I understand it correctly, you're naming it root arg because it adds a root arg to the query and create mutation. But this is actually more: It defines the relation between product and its variants: A variant can't exist without a product. Wouldn't something like `parent: true` be more fitting? 🤔

Reply:
> But this is actually more: It defines the relation between product and its variants:

Not really, the relation is defined in the entity using standard mikro-orm.

> A variant can't exist without a product.

that is correct. BUT it is also correct for the product.category relation (if that would be not nullable)

> Wouldn't something like parent: true be more fitting? 🤔

hm, yes not bad. However `parent` alone, is also not very meaningful.